### DISCUSSION TITLE
TempRes set temperature but not "temperatuer set" correctly

### DISCUSSION THREAD

**Discussion by g6094199** [arendst/Tasmota#21280] [Q&A]
I have a tuya device where the mcu sends  NO decimal places for FnId 71 (temperature) and 72 (temperature set).
The tasmota behavious is one decimal place so the shown temps are wrong.
So I set:

`TempRes 	Temperature sensor resolution
0..3 = maximum number of decimal places`

to 0 und the temerature is now set correct but the temperature set is still wrong

![grafik](https://github.com/arendst/Tasmota/assets/37833326/a9d01d6a-b58b-4467-b589-9d9b9074fff0)

how to set also TempRes for the "temerature set" value?

**Comment by sfromis** [comment]
Try command `TuyaTempSetRes`
